src/components/Dialogs/Dialogs.jsx

```diff
@@ -2,45 +2,61 @@ import React from "react";
 import s from "./Dialogs.module.css";
 import { NavLink } from "react-router-dom";
 
-const DIALOGS = "/dialogs";
-
-const dialogItems = ["Руслана", "Роман", "Тарас", "Сірожа", "Андрій"];
-
 const DialogItem = (props) => {
+  let path = "/dialogs/" + props.id;
+
   return (
     <div className={s.dialog}>
-      <NavLink to={`${DIALOGS}/${props.id}`} activeClassName={s.active}>
+      <NavLink to={path} activeClassName={s.active}>
         {props.name}
       </NavLink>
     </div>
   );
 };
 
-const message = (props) => {
-  return (
-    <div className={s.messages}>
-      <NavLink to={`/${DIALOGS}/${props.id}`} activeClassName={s.active}>
-        {props.name}
-      </NavLink>
-    </div>
-  );
+const Message = (props) => {
+  return <div className={s.dialog}>{props.message}</div>;
 };
 
 const Dialogs = (props) => {
+  let dialogs = [
```

Comment:
```suggestion
  const dialogs = [
```
якщо змінна не змінюється, то використовуємо конст, при цьому при кожному рендері ти будеш наново творити цю змінну, це не ефективно, краще її винести позакомпонент